l10n_ch_hr_payroll/models/hr_contract.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 #
-#  File: hr_contract.py
+#  File: models/hr_contract.py
 #  Module: l10n_ch_hr_payroll
 #
-#  Created by sge@open-net.ch
+#  Created by cyp@open-net.ch
```

Comment:
Probably a wrong copy/paste while updating the headers. 
In fact, the whole module is a common creation. 



Avec mes meilleures salutations Yvon-Philippe Crittin Senior Developper, OpenERP Specialist Open Net Sàrl www.open-net.ch Accélérateur de productivité 59, Rue de l'industrie CH-1030 Bussigny Autoroute A1, sortie Bussigny Bus TL 17, arrêt Croix-de-Plan T: +41 (21) 701 42 45 
----- Mail original -----

De: "Yannick Vaucher" <notifications@github.com> 
À: "OCA/l10n-switzerland" <l10n-switzerland@noreply.github.com> 
Cc: "Open Net Ltd" <info@open-net.ch> 
Envoyé: Mardi 6 Octobre 2015 18:30:20 
Objet: Re: [l10n-switzerland] New: commissions and reimbursements mgt in payslips and contracts (#168) 


In l10n_ch_hr_payroll/models/hr_contract.py : >  #  Module: l10n_ch_hr_payroll
>  #
> -#  Created by sge@open-net.ch
> +#  Created by cyp@open-net.ch 
Was this intended or author was changed because of a bad copy paste? (just making sure) 
— 
Reply to this email directly or view it on GitHub .